### DISCUSSION TITLE
Where are transactions stored?

### DISCUSSION THREAD

**Discussion by nico89-exp** [DeFiCh/app#829] [Q&A] [answered]
Hi @izzycsy. Another question that I would like to ask: where is the transaction history stored? 

Background: I use the app with the same wallet on different computers and have the impression that the transaction history is not identical. It would be good to backup transaction data and copy to the other device.

Thanks.

**Comment by izzycsy** [comment, marked as answer]
Hi @nico89-exp , transactions history are stored in the [blockchain](https://mainnet.defichain.io/#/DFI/mainnet/home), as a user you can click Export Data for each token. 

<img width="790" alt="DFI   Export Data" src="https://user-images.githubusercontent.com/65014479/120486799-f3cb9800-c3e7-11eb-8327-ed9789279b01.png">
